### DISCUSSION TITLE
What happens on the next version when ignoring an update and updating it in a different PR?

### DISCUSSION THREAD

**Discussion by dave-yotta** [renovatebot/renovate#19946] [Request Help] [answered]
Closing the PR says that it will be ignored.  I'd like to do this in situations where the update needs to be opened on a feature branch involing a lot of work - so the version would then be eventually updated on dev when this feature is merged.

Will the closed renovate/some-package-1.2 prevent a future PR of renovate/some-package-2.0 being opened?

Should the closed 1.2 PR have it's name changed as part of the feature merging process?

I tried to search for this but there are a lot of unrelated similar results, sorry.

**Comment by viceice** [comment, marked as answer]
no, it won't. but it would if you close the v2 pr, then renovate suppress any v2 update pr until you manually update to v2 or use the retry checkbox
